duckduckgo/Android
# Update libraries

Task/Issue URL: https://app.asana.com/0/1125189844152671/1200472875632052
Tech Design URL: 
CC: 

**Description**:
This PR updates the following libraries:
1. refreshVersions: 0.9.7 → 0.10.1
1. recyclerview: 1.2.0 → 1.2.1
1. turbine: 0.5.1 → 0.52
1. glide: 4.11.0 → 4.12.0
1. leakcanary: 2.5 → 2.7
1. retrofit: 2.8.1 → 2.9.0
1. mockito-kotlin: 2.1.0 → 2.2.0
1. androidx test: 1.3.0 → 1.4.0-beta02
1. room: 2.2.5 → 2.2.6
1. dagger: 2.32 → 2.37
1. lottie: 3.4.0 → 3.7.0

**Steps to test this PR**:
1. App should build and pass all the tests.
1. Install the app and add some favourites and bookmarks.
1. Check in the bookmarks screen the favicons have been saved
1. Test the different fire button animations and make sure they all work as expected.

Note: All the changes have been tested already on API 21


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
